Sources/TransferMethod/AddTransferMethodPresenter.swift

```diff
@@ -95,7 +95,7 @@ final class AddTransferMethodPresenter {
             else {
                 return
         }
-        var hyperwalletTransferMethod: HyperwalletTransferMethod
```

Comment:
I think we still can leave it because for most of the cases we don't need to initialize a hyperwalletTransferMethod.